### ISSUE TITLE
EZPlaceholderHook deprecated

### ISSUE THREAD

**Issue by SlimeDog** [Rocologo/MobHunting#491]
Spigot 1.13.2 final
PlaceholderAPI 2.10.0-b65
MobHunting 6.5.4-SNAPSHOT-B771

Per [PlaceholderAPI](http://ci.extendedclip.com/job/PlaceholderAPI/changes):
```
#65 (May 5, 2019 10:40:16 AM)
Remove EZPlaceholderHook which has been deprecated for over a year. (commit: 6495a38) — Ryan McCarthy / githubweb
``` 
Consequently, placeholders do not work.

On server start-up
```
[08:03:54] [Server thread/INFO]: ^[[0;33;22m[MobHunting]^[[m Compatibility with ActionAnnouncer is disabled in config.yml^[[m
[08:03:54] [Server thread/INFO]: ^[[0;33;22m[MobHunting] ^[[mEnabling compatibility with PlaceholderAPI (2.10.0-DEV-65).^[[m
[08:03:54] [Server thread/ERROR]: Error occurred while enabling MobHunting v6.5.4-SNAPSHOT-B771 (Is it up to date?)
java.lang.NoClassDefFoundError: me/clip/placeholderapi/external/EZPlaceholderHook
        at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
        at java.lang.ClassLoader.defineClass(ClassLoader.java:1016) ~[?:?]
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:132) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.plugin.java.JavaPluginLoader.getClassByName(JavaPluginLoader.java:200) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:93) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:82) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        at one.lindegaard.MobHunting.compatibility.PlaceholderAPICompat.<init>(PlaceholderAPICompat.java:37) ~[?:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:?]
        at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
        at java.lang.reflect.Constructor.newInstance(Constructor.java:490) ~[?:?]
        at java.lang.Class.newInstance(Class.java:584) ~[?:?]
        at one.lindegaard.MobHunting.compatibility.CompatibilityManager.register(CompatibilityManager.java:50) ~[?:?]
        at one.lindegaard.MobHunting.compatibility.CompatibilityManager.registerPlugin(CompatibilityManager.java:29) ~[?:?]
        at one.lindegaard.MobHunting.MobHunting.onEnable(MobHunting.java:253) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:265) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:347) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:410) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugin(CraftServer.java:436) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugins(CraftServer.java:350) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at net.minecraft.server.v1_13_R2.MinecraftServer.l(MinecraftServer.java:580) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:542) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:420) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:294) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:698) [spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.ClassNotFoundException: me.clip.placeholderapi.external.EZPlaceholderHook
        at java.net.URLClassLoader.findClass(URLClassLoader.java:471) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:136) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:82) ~[spigot-1.13.2.jar-2019-04-22-2230:git-Spigot-1a3504a-a46fdbc]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        ... 29 more
```

**Comment by SlimeDog** [comment]
PlaceholderAPI dev [put it back temporarily](https://github.com/PlaceholderAPI-Expansions/PlaceholderAPI/commit/75c3027e055867f9b1f1f529b743db679f5668e6), but you really need to update.

**Comment by Rocologo** [comment]
Yes, the dev wrote to me today. I will update soon.  I need to release a 1.14 version as well.

**Comment by SlimeDog** [comment]
True that.

**Comment by SlimeDog** [comment]
The method has been replaced temporarily, but on server start-up
```
[08:14:14] [Server thread/WARN]: [PlaceholderAPI] MobHunting is currently using a deprecated method to hook into PlaceholderAPI. This will be disabled after the next update. Please consult [Rocologo] and urge them to update it ASAP.
```

**Comment by Rocologo** [comment]
Fixed in newest release

**Comment by SlimeDog** [comment]
Confirmed fix with MobHunting 6.5.5-b787 on both Spigot 1.13.2 final and Spigot 1.14. Thanks.
